### PR TITLE
WD-13585 refresh  /internet-of-things/management

### PR DESCRIPTION
**Pull request by immortalcodes**
## Done

- refreshed /internet-of-things/management
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [Design](https://www.figma.com/design/CmTm8vAUKUILc3Hu4EzSws/Internet-of-Things?node-id=2-2278&node-type=rounded_rectangle&m=dev)
- [Copydoc](https://docs.google.com/document/d/1C-1ywVzBHkc4I3f-G7Rpy-epaXdbPisbolHUC1Jb7cE/edit)
- [Demo](https://ubuntu-com-14288.demos.haus/internet-of-things/management)
## Issue / Card
[WD-13585](https://warthogs.atlassian.net/browse/WD-13585)

Fixes #

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-13585]: https://warthogs.atlassian.net/browse/WD-13585?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/internet-of-things/management.html [hunk 1]
```diff
@@ -88,126 +110,170 @@ <h3>Integrate with cloud solutions</h3>
...
-                            height="28",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},) | safe
-              }}
-              <h3 class="p-heading-icon__title p-heading--4">Whitepaper</h3>
+      </div>
+      <div class="row--25-75-on-large">
+        <div class="col">
+          <div class="p-equal-height-row has-divider-1">
```
Comment (mtruj013) [inline templates/internet-of-things/management.html:137]: The images in these cols look very large compared to the corresponding section on https://ubuntu.com/desktop which we're basing off of, could we make them smaller?

File: templates/internet-of-things/management.html [hunk 2]
```diff
@@ -88,126 +110,170 @@ <h3>Integrate with cloud solutions</h3>
...
+                  alt="Snap store",
+                  width="92",
+                  height="80",
+                  hi_def=True,
+                  loading="lazy") | safe
+                  }}
+                </div>
+              </div>
+              <div class="p-equal-height-row__item">
+                <h3 class="p-heading--5">Ubuntu Core - Dedicated Snapstore</h3>
```
Comment (mtruj013) [inline templates/internet-of-things/management.html:179]:
```suggestion
                <h3 class="p-heading--5">Ubuntu Core &ndash; Dedicated Snapstore</h3>
```

**Review by mtruj013 — DISMISSED** [review]
Thanks @immortalcodes, some minor comments in the code. Once you remove the whitespaces in the image template please re-run djlint so that they format correctly

**Comment by mattea-turic** [comment]
Not too familiar with patterns prior to rebranding, so bear with me here @immortalcodes 

Looks like there's an extra line at bottom of table here
Screenshot 2024-09-12 at 10 00 11

Logos/Images here look a little big
Screenshot 2024-09-13 at 11 16 46
esp. on mobile. 
Screenshot 2024-09-13 at 11 17 54

**Comment by immortalcodes** [comment]
> Not too familiar with patterns prior to rebranding, so bear with me here @immortalcodes
> 
> Looks like there's an extra line at bottom of table here Screenshot 2024-09-12 at 10 00 11
> 
> Logos/Images here look a little big Screenshot 2024-09-13 at 11 16 46 esp. on mobile. Screenshot 2024-09-13 at 11 17 54

Yes you are right, it's a trade off to get the layout right on medium screens, if I remove the extra lists, it would look something like this on medium screens
image
Further Vanilla [suggests](https://vanillaframework.io/docs/patterns/matrix) that we should have items in multiple of 3 in such matrices.

I will scale down the logos and push the changes.

File: templates/internet-of-things/management.html [hunk 3]
```diff
@@ -88,126 +110,170 @@ <h3>Integrate with cloud solutions</h3>
               </div>
             </div>
           </li>
+          <li class="p-matrix__item"></li>
+          <li class="p-matrix__item"></li>
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply templates/internet-of-things/management.html:114]: https://github.com/canonical/ubuntu.com/pull/14288#issuecomment-2354527979

File: templates/internet-of-things/management.html [hunk 4]
```diff
@@ -88,126 +110,170 @@ <h3>Integrate with cloud solutions</h3>
               </div>
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply templates/internet-of-things/management.html:110]: Yes, you are right, it has been directly edited so I totally missed it.

File: templates/internet-of-things/management.html [hunk 5]
```diff
@@ -88,126 +110,170 @@ <h3>Integrate with cloud solutions</h3>
...
+            <b>We are making Device Management simple and flexible</b>
+          </h2>
+          <blockquote class="p-pull-quote">
+            <p class="p-pull-quote__quote u-no-max-width">
+              In collaboration with Canonical, we are making it simpler to reliably connect devices to Microsoft Azure IoT services. Snap support in Azure IoT Edge helps ensure consistent performance, enhanced security, and efficient updates across Linux distributions that support Snaps.
+            </p>
+            <cite class="p-pull-quote__citation">Kam VedBrat, GM, Azure IoT</cite>
+          </blockquote>
+          <blockquote class="p-pull-quote">
+            <p class="p-pull-quote__quote u-no-max-width">
```
[the remark this answers is not in the document]
Comment (akbarkz) [reply templates/internet-of-things/management.html:237]: @immortalcodes please remove this quote as it was not confirmed in the copy doc yet

File: templates/internet-of-things/management.html (hunk 1 above)
Comment (akbarkz) [reply templates/internet-of-things/management.html:137]: @mattea-turic what's your call on this?

Comment (immortalcodes) [reply templates/internet-of-things/management.html:137]: I have reduced the size of images, let me know if that works.

**Comment by mattea-turic** [comment]
> > Not too familiar with patterns prior to rebranding, so bear with me here @immortalcodes
> > Looks like there's an extra line at bottom of table here Screenshot 2024-09-12 at 10 00 11
> > Logos/Images here look a little big Screenshot 2024-09-13 at 11 16 46 esp. on mobile. Screenshot 2024-09-13 at 11 17 54
> 
> Yes you are right, it's a trade off to get the layout right on medium screens, if I remove the extra lists, it would look something like this on medium screens image Further Vanilla [suggests](https://vanillaframework.io/docs/patterns/matrix) that we should have items in multiple of 3 in such matrices. I could remove them if the layout on medium looks fine to you, that will solve the extra line problem as well.
> 
> I will scale down the logos and push the changes.
> 
> @mattea-turic @mtruj013

My only concern with this layout is that the fourth item could be mistaken for a new section as it isn't obvious that it's part of the first three. Would it be possible to do use wrapping so that we have 4 full width items rather than just 1?

**Comment by immortalcodes** [comment]
So I have figured out a solution for this and implemented the same layout on medium screens, I have checked all other use-cases of same class elsewhere, it should be good to go with and will not break anything.

**Review by akbarkz — APPROVED** [review]

**Pull request merged**